src/main/java/com/_up/megastore/controllers/interfaces/IProductController.java

```diff
@@ -30,4 +30,10 @@ ProductResponse updateProduct(
             @RequestPart @Nullable MultipartFile multipartFile
     );
 
+    @PatchMapping(value = "/{productId}", consumes = { MediaType.MULTIPART_FORM_DATA_VALUE, MediaType.APPLICATION_JSON_VALUE })
+    @ResponseStatus(HttpStatus.OK)
+    ProductResponse deleteProduct(
+            @PathVariable UUID productId
+    );
+
```

Comment:
Siguiendo con el comentario anterior, esto debería ser así

```suggestion
    @DeleteMapping(value = "/{productId}" )
    @ResponseStatus(HttpStatus.OK)
    ProductResponse deleteProduct(
            @PathVariable UUID productId
    );

```